style.css

```diff
@@ -193,16 +280,230 @@ h3 {
   background-color: #1a2236;
 }
 
-/* Aboutme Section css */
+/* Works Desktop Section CSS */
+
+@media (min-width: 768px) {
+  .works {
+    grid-template-columns: 26.8% 19.8% 16.8% 19.8% 16.8%;
+    grid-template-rows: 32.2% 4.8% 10.6% 10.6% 10.6% 15.6% 15.6%;
+    grid-auto-flow: row;
+    gap: 0 0;
+    height: 1660px;
+  }
+
+  .works img {
+    display: none;
+  }
+
+  .buttonproject img {
+    display: block;
+  }
+
+  h3 {
+    font-weight: 600;
+    font-size: 30px;
+  }
+
+  .html {
+    width: 132px;
+  }
+
+  .ruby {
+    width: 169px;
+  }
+
+  .js {
+    width: 144px;
+  }
+
+  .html,
+  .ruby,
+  .js {
+    height: 40px;
+    font-size: 20px;
+    line-height: normal;
+  }
```

Comment:
Hi @medaminedev66,
I need to add these classes to the HTML just because these boxes had a different width on the Figma template, and If I delete this the text that is contained in these boxes is oversized.

Reply:
Hi @AntonioHincapie, try not to fix the width of the boxes, make it `auto`, or `100%`, and try to add the paddings to the boxes. The width of the box will take the width of the language text.